src/main/java/xfacthd/framedblocks/common/compat/jei/FramingSawRecipeCategory.java

```diff
@@ -186,14 +191,32 @@ public void draw(FramingSawRecipe recipe, IRecipeSlotsView slots, GuiGraphics gr
     }
 
     @Override
+    public void getTooltip(ITooltipBuilder tooltip, FramingSawRecipe recipe, IRecipeSlotsView slots, double mouseX, double mouseY)
```

Comment:
I would prefer a helper method for these two methods instead of entirely duplicating the code